src/connectors/pagination/connectPagination.js

```diff
@@ -165,9 +170,12 @@ export default function connectPagination(renderFn, unmountFn) {
 
       getWidgetSearchParameters(searchParameters, { uiState }) {
         const uiPage = uiState.page;
-        if (uiPage)
+
+        if (uiPage) {
           return searchParameters.setQueryParameter('page', uiState.page - 1);
-        return searchParameters.setQueryParameter('page', 0);
+        }
+
+        return searchParameters.setQueryParameter('page');
```

Comment:
this implies we no longer need `setPage` on the helper right?

Reply:
We never _really_ have the need for `setPage` since it forwards the call. The removal of those methods could be part of the process to remove the unused methods of the Helper.